### PR TITLE
Added support for binding to all Style properties directly.

### PR DESCRIPTION
**Pull request by hamorphis**

### CODE REVIEW

**Comment by nsndeck** [comment]
:+1:

**Comment by enchev** [comment]
:100:

File: ui/core/view-common.ts [hunk 1]
```diff
@@ -825,14 +907,14 @@ export class View extends proxy.ProxyObject implements definition.View, definiti
...
     applyXmlAttribute(attributeName: string, attributeValue: any): boolean {
-        if (attributeName === "margin") {
-            this.style.margin = attributeValue;
-            return true;
-        }
-        else if (attributeName === "padding") {
-            this.style.padding = attributeValue;
-            return true;
-        }
+        //if (attributeName === "margin") {
```
Comment (hshristov) [inline ui/core/view-common.ts]: You can delete this code

**Comment by ErjanGavalji** [comment]
:+1:

**Comment by hshristov** [comment]
:+1:

**Pull request merged**